dbt-bigquery/src/dbt/adapters/bigquery/credentials.py

```diff
@@ -17,10 +19,13 @@
 from dbt.adapters.contracts.connection import Credentials
 from dbt.adapters.events.logging import AdapterLogger
 from dbt.adapters.exceptions.connection import FailedToConnectError
+from dbt.adapters.bigquery.auth_providers import create_token_service_client
 
 
 _logger = AdapterLogger("BigQuery")
 
+_TOKEN_FILE_PATH = "/tmp/access_token.json"
```

Comment:
id this secure? Should it be encrypted? Is this only in our sandbox so it's fine? (just asking questions)